build/Makefile

```diff
@@ -428,29 +430,43 @@ ensure-windows-buildx:
 	#
 	# Windows image builds must be directed to a specific buildx context.
 	# The default context does not support building cross platform images.
-	-DOCKER_CLI_EXPERIMENTAL=enabled docker buildx create --name=$(BUILDX_WINDOWS_BUILDER) --use
+	-docker buildx create --name=$(BUILDX_WINDOWS_BUILDER) 
```

Comment:
https://github.com/docker/docker-ce-packaging/pull/490 I found that.
I guess buildx support were out of experimental since 20.10.0 but there is info into release note